diff --git a/flora2/cc/cc_config.P b/flora2/cc/cc_config.P
index 9d296e5..65fd86b 100644
--- a/flora2/cc/cc_config.P
+++ b/flora2/cc/cc_config.P
@@ -97,7 +97,7 @@
 
     %% If Module isn't compiled or is old --- recompile
     ;   xsb_configuration(compiler, CC),
-	consult(Module, [cc(CC), cc_opts(' ')]),
+	consult(Module, [cc(CC), cc_opts(' -fPIC ')]),
 	flora_file_op(rename,Objfile, TrgPrlgObjFile),
 	( % MacOS
 	flora_running_under(darwin)
